python/docs/languages/c-api.rst

```diff
@@ -7,7 +7,9 @@ Core API
 --------
 
 The following are the C functions exposed by the BridgeStan library in ``bridgestan.h``.
-These are wrapped in the various high-level interfaces.
+These are wrapped in the various high-level interfaces. An example calling
+these functions from pure-C can be found in the ``c-example/`` subdirectory
```

Comment:
I don't know the .rst format.  I did try to track down how-it-works, but I don' tknow how to verify this link works, assuming the single back tick is a link.